zeton/templates/index_child.html

```diff
@@ -39,23 +40,21 @@ <h2 style="color: green;">Możesz korzystać z komputera</h2>
                 {% endfor %}
             </table>
             </ul>
-            <center><h5>SZKOLNY REKORD TYGODNIA--> chipsy & extra 100 pkt</h5></center>
+
+            <h5 class="text-center">SZKOLNY REKORD TYGODNIA--> chipsy & extra 100 pkt</h5>
         </div>
 
         <div id="odejmowanie" class="col-xl-6 square">
-            <center><h2>Wykorzystaj punkty</h2>
-                <form action="/api/child/{{ child.id }}/points/use" method="post">
-                    <div class="form-group button">
-                        Liczba punktów: <input class="form-control" type="number"
-                                               name="points"
-                                               required>
+            <h2 class="text-center">Wykorzystaj punkty</h2>
 
-                        <input class="btn btn-info" type="submit" name="wykorzystaj punkty"
-                               value="Wykorzystaj punkty"/>
-                    </div>
-                    <h4>NAGRODY</h4>
-                </form>
-            </center>
+            <form action="/api/child/{{ child.id }}/points/use" method="post">
```

Comment:
Po wykorzystaniu punktów nie pozostajemy na tej samej stronie.

Reply:
jeśli zamienimy na:

`           <form action="/api/child/{{ child.id }}/points/use?return_url=/child/{{ child.id }}" method="post">`

to działa